crates/ty_ide/src/hover.rs

```diff
@@ -362,6 +656,32 @@ mod tests {
         ");
     }
 
+    #[test]
+    fn hover_module_import() {
+        let mut test = cursor_test(
+            r#"
+            import li<CURSOR>b
+
+            lib
+            "#,
+        );
+
+        test.write_file(
+            "lib.py",
+            r"
+        '''
+        The cool lib_py module!
```

Comment:
This one's even worse and produces no result because apparently hovering an import doesn't yield the module (even though I fixed goto-def on it in a previous PR). I assume this is because hover currently insists it needs the type of the hovered item, not the def/decl.